pkg/comp-functions/functions/vshn-postgres-func/mailgun_alerting_test.go

```diff
@@ -0,0 +1,51 @@
+package vshnpostgres
+
+// test cases for the function MailgunAlerting
+
+import (
+	"context"
+	"testing"
+
+	alertmanagerv1alpha1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1alpha1"
+	"github.com/vshn/appcat/pkg/comp-functions/runtime"
+)
+
+func init() {
+	err := runtime.AddToScheme(alertmanagerv1alpha1.SchemeBuilder)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func TestMailgunAlerting(t *testing.T) {
+	ctx := context.Background()
+
+	// return Normal when there is no email configured
+	inputFnio := loadRuntimeFromFile(t, "alerting/06-GivenNoEmail.yaml")
+
+	result := MailgunAlerting(ctx, inputFnio)
+
+	if result.Resolve().Severity != "Normal" {
```

Comment:
Nitpick: Using the `assert` library would make this test more readable than these if statements.